java/ql/src/utils/flowtestcasegenerator/GenerateFlowTestCase.py

```diff
@@ -207,11 +217,32 @@ def copyfile(fromName, toFileHandle):
 
 with open(resultQl, "w") as f:
     copyfile("testHeader.qlfrag", f)
-    if len(supportModelRows) != 0:
-        copyfile("testModelsHeader.qlfrag", f)
-        f.write(", ".join('"%s"' %
-                          modelSpecRow[0].strip() for modelSpecRow in supportModelRows))
-        copyfile("testModelsFooter.qlfrag", f)
+
+if len(supportModelRows) != 0:
```

Comment:
NIT: this could be a bit more pythonic.

```suggestion
if supportModelRows:
```

Reply:
If it is ok with you, I would prefer to leave it as it is. Basically that is a cut and paste from before and for someone (like me) that is not a python expert it becomes a bit more readable in this way.